### DISCUSSION TITLE
Purge content `*.html` except `*.not.html`

### DISCUSSION THREAD

**Discussion by zhendalf** [FullHuman/purgecss#794] [Q&A] [answered]
How do I define the `content` property that would allow to include all `*.html` files, but ignore `*.not.html` files?

**Comment by Ffloriel** [comment, marked as answer]
Take a look at the options here: https://purgecss.com/configuration.html#options
The one you are looking for should be `skippedContentGlobs `, you could have something like
```js
{
    content: ['*.html'],
    skippedContentGlobs: ['*.not.html']
}
```

**Reply by zhendalf** [reply]
Nice! I missed this configuration option. Thank you for pointing it out. 

However, I was able to do what I needed with the following glob:
```js
{
  content: ['!(*.not).html'],
}
```
